presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/clientproviders/RetryingPrestoThriftServiceProvider.java

```diff
@@ -96,11 +101,13 @@ public PrestoThriftService selectedHostClient(List<HostAddress> hosts)
         private final Supplier<PrestoThriftService> clientSupplier;
         private final RetryDriver retry;
         private PrestoThriftService client;
+        private RetryingPrestoThriftServiceStats retryingPrestoThriftServiceStats;
 
-        public RetryingService(Supplier<PrestoThriftService> clientSupplier, RetryDriver retry)
+        public RetryingService(Supplier<PrestoThriftService> clientSupplier, RetryDriver retry, RetryingPrestoThriftServiceStats stats)
         {
             this.clientSupplier = requireNonNull(clientSupplier, "clientSupplier is null");
             this.retry = retry.onRetry(this::close);
+            this.retryingPrestoThriftServiceStats = requireNonNull(stats, "stats is null");
```

Comment:
You changed the name of a parameter, but not of the field. Can you also check other places where having a shorter name is preferred?